WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
```

Comment:
Chooser Dialog is not limited to ChooseFolder dialog. Other dialogs like : ZmChooseAccountDialog, ZmPickTagDialog also uses same class. Better to name the properties generically . Also great if we consider the styling applied to be working fine for all the dialogs and those would also be affected with the change.